pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -242,8 +242,10 @@ public void runSegmentMetrics() {
       // Synchronization provided by Controller Gauge to make sure that only one thread updates the gauge
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.NUMBER_OF_REPLICAS,
           nReplicasExternal);
-      _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_OF_REPLICAS,
-          nReplicasExternal*100/nReplicasIdealMax);
+      if (nReplicasIdealMax != 0) {
+        _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_OF_REPLICAS,
```

Comment:
We won't emit a value for nReplicasIdealMax = 0. Seems there should be one. What happens if we've alerts on this and no value is emitted for a while ?